### PR TITLE
spidev: process arguments for raw command

### PR DESCRIPTION
**Pull request by rminnich**
for the raw command, for each arg, dump arg size bytes to stdout.

Ryan, feel free to tell me this should be done differently/better :-)

I just did not want to lose these changes.

### CODE REVIEW

File: cmds/fwtools/spidev/spidev_linux.go [hunk 1]
```diff
@@ -69,10 +70,16 @@ func run(args []string, spiOpen spiOpenFunc, input io.Reader, output io.Writer)
 		return fmt.Errorf("%w:%v", errFlag, err)
 	}
 
-	if fs.NArg() != 1 {
-		return fmt.Errorf("%w: %s <raw|sfdp|id>", errCommand, fs.FlagUsages())
+	if fs.NArg() == 0 {
+		return fmt.Errorf("%w: %s <raw [bytes]...|sfdp|id>", errCommand, fs.FlagUsages())
```
Comment (10000TB) [inline cmds/fwtools/spidev/spidev_linux.go:74]: would it make sense to update lin 68 to be consistent with this new format ? 

line 68: `return fmt.Errorf("%w:<raw|sfdp|id>", errCommand)`

Comment (10000TB) [reply cmds/fwtools/spidev/spidev_linux.go:74]: ah, line 80 looks like matching  68, but mis-match with here. does it make sense to use one form to be consistent ?

**Review by 10000TB — DISMISSED** [review]

**Comment by 10000TB** [comment]
BTW, not related to this change, but `pflag` seems to print out an error on `spidev --help` , which looks weird

```
xuehaohu@worldpeace:~$ spidev --help
Usage of spidev:
  -D, --device string   spidev device (default "/dev/spidev0.0")
  -s, --speed uint32    max speed in Hz (default 500000)
2024/05/15 13:04:58 Error: pflag: help requested
xuehaohu@worldpeace:~$
```

**Comment by rminnich** [comment]
Hi, I cleaned this up a bit, including getting rid of spf13 and the spew from the flag package. PTAL.

**Review by binjip978 — APPROVED** [review]

**Pull request merged**